privacyidea/api/before_after.py

```diff
@@ -191,22 +196,7 @@ def before_request():
                         "action_detail": "",
                         "info": ""})
 
-    if g.logged_in_user.get("role") == "user":
-        # A user is calling this API
-        # In case the token API is called by the user and not by the admin we
-        #  need to restrict the token view.
-        CurrentUser = get_user_from_param({"user":
-                                               g.logged_in_user.get(
-                                                   "username"),
-                                           "realm": g.logged_in_user.get(
-                                               "realm")})
-        request.all_data["user"] = CurrentUser.login
-        request.all_data["resolver"] = CurrentUser.resolver
-        request.all_data["realm"] = CurrentUser.realm
-        g.audit_object.log({"user": CurrentUser.login,
-                            "resolver": CurrentUser.resolver,
```

Comment:
There is a bug here: Because this line is removed, the login name, resolver and realm aren't written to the audit log anymore if the current blueprint isn't ``token_blueprint``.

Reply:
You are right. Since the code was simplified the auditlog is not written earlier. 
So we can simply drop the ``if "token_blueprint`` condition.

Reply:
all done in 916c01da

Reply:
:+1:

Reply:
:+1: